<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>QGenLab @ UNLV</title>
  <meta content="Dr. <b> Qian Liu </b>'s Computational Genomics Lab" name="description">
  <meta content="Computational Genomics, Deep Learning, Long-read Analysis" name="keywords">

  <!-- Favicons -->
  <link href="assets/img/favicon.png" rel="icon">
  <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon">

  <!-- Google Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Raleway:300,300i,400,400i,500,500i,600,600i,700,700i|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i" rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/layout/animate.css/animate.min.css" rel="stylesheet">
  <link href="assets/layout/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/layout/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href="assets/layout/boxicons/css/boxicons.min.css" rel="stylesheet">
  <link href="assets/layout/glightbox/css/glightbox.min.css" rel="stylesheet">
  <link href="assets/layout/remixicon/remixicon.css" rel="stylesheet">
  <link href="assets/layout/swiper/swiper-bundle.min.css" rel="stylesheet">

  <!-- Template Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">

</head>

<body>

  <!-- ======= Header ======= -->
  <header id="header" class="fixed-top d-flex align-items-center">
    <div class="container d-flex align-items-center">

      <a href="index.html" class="logo me-auto"><img src="Qgenlab-logo.jpg" alt="" class="img-fluid"></a>

      <nav id="navbar" class="navbar">
        <ul>
          <li><a href="index.html" class="active">Home</a></li>
          <li><a href="research.html">Research</a></li>
          <li><a href="publications.html">Publications</a></li>
          <li><a href="softwares.html">Softwares</a></li>
          <li><a href="team.html">Team</a></li>
          <!-- <li><a href="contact.html">Contact</a></li>-->
        </ul>
        <i class="bi bi-list mobile-nav-toggle"></i>
      </nav><!-- .navbar -->

    </div>
  </header><!-- End Header -->

  <main id="main">

<br></br>

<!-- ======= Features Section ======= -->
<section id="features" class="features">
  <div class="container">

    <div class="section-title">
      <h2>Publications</h2>
    </div>

    <div class="row">
      <div class="col-lg-3">
        <ul class="nav nav-tabs flex-column">
          <li class="nav-item">
            <a class="nav-link active show" data-bs-toggle="tab" href="#tab-0">Featureed Publications</a>
          </li>

          <li class="nav-item">
            <a class="nav-link" data-bs-toggle="tab" href="#tab-1">Modification Detection</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" data-bs-toggle="tab" href="#tab-2">Genome Analysis</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" data-bs-toggle="tab" href="#tab-3">Transcriptome Analysis</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" data-bs-toggle="tab" href="#tab-4">Protein binding Analysis</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" data-bs-toggle="tab" href="#tab-5">Data Mining</a>
          </li>
        </ul>
      </div>
      <div class="col-lg-9 mt-4 mt-lg-0">
        <div class="tab-content">
          <div class="tab-pane active show" id="tab-0">
          <sup>&#10013;</sup>: corresponding authors; <sup>&#9913;</sup>: co-first author <br>
          <div class="row">
          <ol>
              <li> Umair Ahsan, <b> Qian Liu <sup>&#9913;</sup></b>, Li Fang, Kai Wang. NanoCaller for accurate detection of SNPs and indels from long-read sequencing by deep neural networks. <i><u>Genome Biology</i></u> 2021; 22:261. </li>
              <li> Daniela Georgieva, <b> Qian Liu <sup>&#9913;</sup></b>, Kai Wang, Dieter Egli. Detection of Base Analogs Incorporated During DNA Replication by Nanopore Sequencing. <i><u>Nucleic Acids Research</i></u> 2020; gkaa517. </li>
              <li> <b> Qian Liu </b>, Li Fang, Guoliang Yu, Depeng Wang, Chuan-Le Xiao, Kai Wang. Detection of DNA base modifications by deep recurrent neural network on Oxford Nanopore sequencing data. <i><u>Nat. Commun</i></u> 2019; 10 (1), 1-11. </li>
              <li> <b> Qian Liu </b>, Peng Zhang, Depeng Wang, Weihong Gu, Kai Wang. Interrogating the "unsequenceable" genomic trinucleotide repeat disorders by long-read sequencing. <i><u>Genome Medicine</i></u> 2017; 9(1):65. </li>
          </ol>
          </div>
        </div>
          <div class="tab-pane" id="tab-1">
            <sup>&#10013;</sup>: corresponding authors; <sup>&#9913;</sup>: co-first author <br>
            <div class="row">
            <ol>
                <li> Daniela Georgieva, <b> Qian Liu <sup>&#9913;</sup></b>, Kai Wang, Dieter Egli. Detection of Base Analogs Incorporated During DNA Replication by Nanopore Sequencing. <i><u>Nucleic Acids Research</i></u> 2020; gkaa517. </li>
                <li> <b> Qian Liu </b>, Li Fang, Guoliang Yu, Depeng Wang, Chuan-Le Xiao, Kai Wang. Detection of DNA base modifications by deep recurrent neural network on Oxford Nanopore sequencing data. <i><u>Nat. Commun</i></u> 2019; 10 (1), 1-11. </li>
                <li> <b> Qian Liu </b>, Daniela C. Georgieva, Dietrich M. Egli, Kai Wang. NanoMod: a computational tool to detect DNA modifications using Nanopore long-read sequencing data. <i><u>BMC genomics</i></u> 2019; 20 (1), 31-42. </li>
            </ol>
            </div>
          </div>

          <div class="tab-pane" id="tab-2">
            <sup>&#10013;</sup>: corresponding authors; <sup>&#9913;</sup>: co-first author <br>
            <div class="row">
              <ol>
                  <li> <b> Qian Liu </b>, Li Fang, Alex Mas Monteys, Beverly L. Davidson, Kai Wang. Direct detection and quantification of short tandem repeats on ionic signal data from Nanopore sequencing. Under review. </li>
                  <li> <b> Qian Liu </b>, Yao Tong, Kai Wang. Genome-wide detection of short tandem repeat expansions by long-read sequencing. <i><u>BMC Bioinformatics</i></u> 2020; 21:542 </li>
                  <li> Zeng, Mei-yun Zhang, Xue-jing Wang, Zheng-mao Hu, Jin-chen Li, Nan Li, Jun-ling Wang, Fan Liang, Qi Yang, <b> Qian Liu </b>, Li Fang, Jun-wei Hao, Fu-dong Shi, Xue-bing Ding, Jun-fang Teng, Xiao-meng Yin, Hong Jiang, Wei-ping Liao, Jing-yu Liu, Kai Wang, Kun Xia, Bei-sha Tang. Long-read sequencing identified intronic repeat expansions in SAMD12 from Chinese pedigrees affected with familial cortical myoclonic tremor with epilepsy. <i><u>J Med Genet.</i></u> 2019; 56 (4), 265-270. </li>
                  <li> <b> Qian Liu </b>, Lingling Shi, Kai Wang. Ethnicity-Specific Reference Genome Assembly by Long-Read Sequencing. <i><u>J Mol Genet Med</i></u> 2018; 12(4), 385. </li>
                  <li> Atlas Khan, <b> Qian Liu </b>, Kai Wang. iMEGES: integrated mental-disorder GEnome score by deep neural network for prioritizing the susceptibility genes for mental disorders in personal genomes. <i><u>BMC bioinformatics</i></u> 2018; 19(17), 501. </li>
                  <li> Xinhua Fu, Jingjing Li, Yu Tian, Weipeng Quan, Shu Zhang, <b> Qian Liu </b>, Fan Liang, Xinlei Zhu, Liangsheng Zhang, Depeng Wang, Jiang Hu. Long-read sequence assembly of the firefly Pyrocoelia pectoralis genome. <i><u>GigaScience</i></u> 2017; 6(12):1–7. </li>
                  <li> <b> Qian Liu </b>, Peng Zhang, Depeng Wang, Weihong Gu, Kai Wang. Interrogating the "unsequenceable" genomic trinucleotide repeat disorders by long-read sequencing. <i><u>Genome Medicine</i></u> 2017; 9(1):65. </li>
              </ol>
          </div>
          </div>

          <div class="tab-pane" id="tab-3">
            <sup>&#10013;</sup>: corresponding authors; <sup>&#9913;</sup>: co-first author <br>
            <div class="row">
              <ol>
                  <li> <b> Qian Liu </b>, Yu Hu, Andres Stucky, Li Fang, Jiang F. Zhong, Kai Wang. LongGF: computational algorithm and software tool for fast and accurate detection of gene fusion by long-read transcriptome sequencing. <i><u>BMC Genomics</i></u> 2020; 21:793 </li>
                  <li> Atlas Khan, <b> Qian Liu <sup>&#9913;</sup></b>, Yunjing Zeng, Andres Stucky, Parish P. Sedghizadeh, Daniel Adelpour, Xi Zhang, Kai Wang, Jiang F. Zhong. Detection of human papillomavirus in cases of head and neck squamous cell carcinoma by RNA-seq and VirTect. <i><u>Mol Oncol.</i></u> 2019; 13 (4), 829-839. </li>
                  <li> Renhua Song, <b> Qian Liu </b>, Jinyan Li: Connecting rules from paired miRNA and mRNA expression data sets of HCV patients to detect both inverse and positive regulatory relationships. <i><u>BMC Genomics</i></u> 2015; 16(Suppl 2):S11. </li>
                  <li> Renhua Song, <b> Qian Liu </b>, Gyorgy Hutvagner, Hung Nguyen, Kotagiri Ramamohanarao, Limsoon Wong, Jinyan Li: Rule discovery and distance separation to detect reliable miRNA biomarkers for the diagnosis of lung squamous cell carcinoma. <i><u>BMC Genomics</i></u> 2014; 15(Suppl 9):S16. </li>
              </ol>
            </div>
          </div>
          <div class="tab-pane" id="tab-4">
            <sup>&#10013;</sup>: corresponding authors; <sup>&#9913;</sup>: co-first author <br>
            <div class="row">
              <ol>
                  <li> <b> Qian Liu </b>, Jing Ren, Jiangning Song, Jinyan Li. Co-occurring atomic contacts for the characterization of protein binding hot spots. <i><u>PLoS ONE</i></u> 2016; 10(12): e0144486. </li>
                  <li> Jing Ren, <b> Qian Liu </b>, John Ellis and Jinyan Li: Prediction of conformational B cell epitopes from positive and unlabeled samples. <i><u>BMC Bioinformatics</i></u> 2015; 16(Suppl 18): S12. </li>
                  <li> <b> Qian Liu </b>, Zhenhua Li, Jinyan Li: Use B-factor related features for accurate classification between protein binding interfaces and crystal packing contacts. <i><u>BMC Bioinformatics</i></u> 2014; Suppl 12, S15. </li>
                  <li> Jing Ren, <b> Qian Liu </b>, John Ellis, Jinyan Li: Tertiary structure-based prediction for conformational B-cell epitopes through B factors. <i><u>Bioinformatics</i></u> 2014; 30(12): i264–i273. </li>
                  <li> <b> Qian Liu </b>, Steven C. H. Hoi, Chee-Keong Kwoh, Limsoon Wong, Jinyan Li: Integrating water exclusion theory into β contacts to predict binding free energy changes and binding hot spots. <i><u>BMC bioinformatics</i></u> 2014; 15(1), 57. </li>
                  <li> <b> Qian Liu </b>, Yi-Ping Phoebe Chen, Jinyan Li: k-partite cliques of protein interactions: A novel subgraph topology for functional coherence analysis on PPI networks. <i><u>Journal of Theoretical Biology</i></u> 2014; 340, 146–154. </li>
                  <li> <b> Qian Liu </b>, Chee-Keong Kwoh, Jinyan Li: Binding affinity prediction for protein-ligand complexes based on β contacts and B factor. <i><u>Journal of Chemical Information and Modeling</i></u> 2013; 53(11), 3076–3085. </li>
                  <li> Zhenhua Li, Ying He, <b> Qian Liu </b>, Liang Zhao, Limsoon Wong, CK Kwoh, H Nguyen, Jinyan Li: Structural analysis on mutation residues and interfacial water molecules for human TIM disease understanding. <i><u>BMC Bioinformatics</i></u> 2013; 14(16), 1–15. </li>
                  <li> <b> Qian Liu </b>, Chee-Keong Kwoh, Steven C. H. Hoi: β atomic contacts: Identifying critical specific contacts in protein binding interfaces. <i><u>PLOS ONE</i></u> 2013; 8(4), e59737. </li>
                  <li> <b> Qian Liu </b>, Limsoon Wong, Jinyan Li: Z-score biological significance of binding hot spots of protein interfaces by using crystal packing as the reference state. <i><u>Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics</i></u> 2012; 1824(12):1457–1467. </li>
                  <li> <b> Qian Liu </b>, Steven C. H. Hoi, Su T. T. Chinh, Zhenhua Li, Chee-Keong Kwoh, Limsoon Wong and Jinyan Li: Structural analysis of the hot spots in the binding between H1N1 HA and the 2D1 antibody: Do mutations of H1N1 from 1918 to 2009 affect much on this binding? <i><u>Bioinformatics</i></u> 2011; 27(18):2529–2536. </li>
                  <li> <b> Qian Liu </b>, Jinyan Li: Protein binding hot spots and the residue-residue pairing preference: A water exclusion perspective. <i><u>BMC Bioinformatics</i></u> 2010; 11:244. </li>
                  <li> <b> Qian Liu </b>, Jinyan Li: Propensity vectors of low-ASA residue pairs in the distinction of protein interactions. <i><u>Proteins: Structure, Function and Bioinformatics</i></u> 2009; 78(3):589–602. </li>
                  <li> Jinyan Li, <b> Qian Liu </b>: Double water exclusion: A hypothesis refining the O-ring theory for the hot spots at protein interfaces. <i><u>Bioinformatics</i></u> 2009; 25(6):743–750. </li>
                  <li> <b> Qian Liu </b>, Jinyan Li. Protein Binding Interfaces and Their Binding Hot Spot Prediction: A Survey. <i><u>Bioinformatics for Diagnosis. Prognosis and Treatment of Complex Diseases</i></u>, 79–106, 2013. </li>
                  <li> <b> Qian Liu </b>, Yi-Ping Phoebe Chen, Jinyan Li: High functional coherence in k-partite protein cliques of protein interaction networks. <i><u>in Bioinformatics and Biomedicine, 2009. BIBM'09. IEEE International Conference</i></u> on. 2009; 111–117. </li>
              </ol>
            </div>
          </div>
          <div class="tab-pane" id="tab-5">
            <sup>&#10013;</sup>: corresponding authors; <sup>&#9913;</sup>: co-first author <br>
            <div class="row">
              <ol>
                  <li> <b> Qian Liu </b>, Shameek Ghosh, Jinyan Li, Limsoon Wong, Kotagiri Ramamohanarao. Discovering pan-correlation patterns from time course data sets by efficient mining algorithms. <i><u>Computing</i></u> 2018; 100(4):421–437. </li>
                  <li> <b> Qian Liu </b>, Renhua Song and Jinyan Li: Inference of gene interaction networks using conserved subsequential patterns from multiple time course gene expression datasets. <i><u>BMC Genomics</i></u> 2015; 16(Suppl 12):S4. </li>
                  <li> Zhiguo Gong, <b> Qian Liu </b>: Improving Keyword Based Web Image Search with Visual Feature Distribution and Term Expansion. <i><u>Knowledge and Information Systems</i></u> 2009; 21(1):113–132. </li>
                  <li> <b> Qian Liu </b>, Jinyan Li, Limsoon Wong, Kotagiri Ramamohanarao: Efficient mining of pan-correlation patterns from time course data. <i><u>in Lecture Notes in Computer Science (including subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics)</i></u>2016; 10086 LNAI pp. 234–249. </li>
                 <li> Jinyan Li, <b> Qian Liu </b>, Tao Zeng: Negative correlations in collaboration: concepts and algorithms. <i><u>in Proceedings of the 16th ACM SIGKDD international conference on Knowledge discovery and data mining</i></u> 2010; 463–472. </li>
                 <li> Z Gong, J Zhang, <b> Qian Liu </b>: Automatic hidden Web database classification. <i><u>in Knowledge Discovery in Databases: PKDD</i></u> 2007; 454–461. </li>
                 <li> Z Gong, <b> Qian Liu </b>, J Zhang: Web image retrieval refinement by visual contents. <i><u>in Advances in Web-Age Information Management</i></u> 2006; 134–145. </li>
                 <li> Z Gong, <b> Qian Liu </b>, J Zhang: Automatic image annotation by mining the Web. <i><u>in Data warehousing and knowledge discovery</i></u> 2006; 449–458. </li>
                 <li> Z Gong, J Zhang, <b> Qian Liu </b>: Hidden-Web database exploration. <i><u>in Intelligent Systems Design and Applications, 2006. ISDA'06. Sixth International Conference</i></u> on. 2006; 838–843. </li>
              </ol>
            </div>
          </div>
        </div>
      </div>
    </div>

  </div>
</section><!-- End Features Section -->

  </main><!-- End #main -->

  <!-- ======= Footer ======= -->
  <footer id="footer">
      <div class="container">
        <div class="copyright">
          &copy; Copyright <strong><span>QGenLab</span></strong>. All Rights Reserved
        </div>
      </div>
    </footer><!-- End Footer -->

  <a href="#" class="back-to-top d-flex align-items-center justify-content-center"><i class="bi bi-arrow-up-short"></i></a>

  <!-- Vendor JS Files -->
  <script src="assets/layout/bootstrap/js/bootstrap.bundle.min.js"></script>
  <script src="assets/layout/glightbox/js/glightbox.min.js"></script>
  <script src="assets/layout/isotope-layout/isotope.pkgd.min.js"></script>
  <script src="assets/layout/php-email-form/validate.js"></script>
  <script src="assets/layout/swiper/swiper-bundle.min.js"></script>
  <script src="assets/layout/waypoints/noframework.waypoints.js"></script>

  <!-- Template Main JS File -->
  <script src="assets/js/main.js"></script>

</body>

</html>
